cmake/GitUtils.cmake

```diff
@@ -0,0 +1,228 @@
+cmake_minimum_required(VERSION 3.14.0)
```

Comment:
This file is taken from somewhere else right? We should link the source in this case, for instance in a comment at the top of this file

Reply:
I have just updated the files with appropriate reference and reporting its license.